PURL-TYPES.rst

```diff
@@ -578,6 +578,25 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+nix
+---
+
+``nix`` for Nix derivations:
+
+- The default package repository is <https://github.com/NixOS/nixpkgs> but the ``repository`` qualifier can override the package repository.
+- The ``name`` is the package ``pname`` field of the given derivation. It is case sensitive.
+- The ``version`` is the ``version`` field of the given derivation.
+- The ``drvpath`` qualifier is the derivation path (``.drvPath``).
+- The ``output`` qualifier is the output field, by default: ``out``
```

Comment:
```suggestion
- The ``output`` qualifier is the name of the output, which helps to distinguish in the case of multi-output derivations
```

The "default" behavior is confusing. It doesn't describe if it should always be set, or if it is "out" in case the field is not present.

Or should it always be set to something, and the fact it's "out" in single-output derivations is an implementation detail of Nix that should be left out of the spec? I prefer this one, which is my the suggestion uses this variant.

I don't think it should be omitted in case of single-output derivations, it'd be odd if it suddenly would appear if an expression is refactored to multiple outputs.

Reply:
including the output field also accounts for if the default is changed in the future and shouldn't be very painful to just include

Reply:
we should just map this to `outputName` of the `drv`.    Then this works with `lib.getLib` etc